### PR TITLE
TS Linter: lint all ts/tsx files

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

**Comment by NickMitrokhin** [comment]
spell-checker complains to expect.eql method.

**Comment by churkin** [comment]
> spell-checker complains to expect.eql method.

@NickMitrokhin This rule already added to the global config

**Review by NickMitrokhin — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -158,7 +158,7 @@
   "scripts": {
     "lint": "npm-run-all -p -c lint-js lint-css",
```
Comment (San4es) [inline package.json]:
```suggestion
    "lint": "npm-run-all -p -c lint-js lint-css lint-ts",
```
To enable CI for TS files. See:
https://github.com/DevExpress/DevExtreme/blob/988321cdedd1a9fc1a22ad598ca8eb101338b36a/docker-ci.sh#L10-L13

File: package.json [hunk 2]
```diff
@@ -158,7 +158,7 @@
...
     "lint-js": "eslint .",
```
Comment (San4es) [inline package.json]: Why this script validates `.tsx` files?
image

File: .lintstagedrc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
     "*.js": ["eslint"],
     "*.{css,less}": ["stylelint"],
-    "*.{ts,tsx}": ["npm run lint-ts"]
+    "*.{ts,tsx}": ["eslint --ext .tsx,.ts"]
```
Comment (San4es) [inline .lintstagedrc]:
```suggestion
    "*.{ts,tsx}": ["eslint"]
```
No need to pass a file extension parameter - you already did this via `"*.{ts,tsx}"`

**Review by San4es — APPROVED** [review]

**Review by vconst — APPROVED** [review]

**Pull request merged**